Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
Turns out I'm an idiot. I forgot to tuck the `curl` check inside a `begin` block and that was causing the failure. This should actually fix Tim's concern earlier as well, because `curl` will fail if the server doesn't respond and that'll blow up the test, so we *should* catch future `icu4c` bumps needed - At least, that's the hope. Pushing now.